5.1.I_dont_have_vinaigrette.py

```diff
@@ -0,0 +1,63 @@
+import re
+from datetime import datetime, timedelta
+from random import random
+
+DATE_FORMAT = r"^\d{4}/\d{2}/\d{2}$"
+DATE_MSG = "The date is not valid. The format should be YYYY/MM/DD"
+MONDAY = 0
+
+
+def check_range(num: int, min: int, max: int) -> bool:
+    return min <= num <= max
+
+
+def is_valid_date(date: str) -> bool:
+    """Check if the date is valid in the format YYYY/MM/DD
+    :param date: date in the format YYYY/MM/DD
+    :return: True if the date is valid
+    """
+    [year, month, day] = date.split("/")
+    return check_range(int(month), min=1, max=12) and check_range(int(day), min=1, max=31)
+
+
+def rand_date_between(start_date: str, end_date: str) -> str:
+    """Generate a random date between two dates
+    :param start_date: start date
```

Comment:
These lines of documentation are redundant, either explain in a sentence or dont explain at all